verticapy/plotting/_plotly/bar.py

```diff
@@ -43,7 +43,9 @@ def _compute_method(self) -> Literal["1D"]:
     def _init_style(self) -> None:
         self.init_trace_style = {"marker_color": self.get_colors(idx=0)}
         self.init_layout_style = {
-            "width": 200 * len(self.layout["labels"]),
+            "yaxis_title": "density",
```

Comment:
I don't think it is always the density. The user can also use customised aggregations. Please check out all the layout params.

Reply:
Done. Now i replaced it with self.layout['method'].